nixos/modules/services/security/authelia.nix

```diff
@@ -379,6 +380,15 @@ in
               Do not include raw secrets in nix settings.
             '';
           }
+          # Removed options. Cannot use mkRemovedOptionModule (or similar methods) due to errors.
+          # See https://github.com/NixOS/nixpkgs/pull/299309#issuecomment-2038240493
+          {
+            assertion = !((instance.settings.server ? host) || (instance.settings.server ? port));
```

Comment:
So I'll again ask you to please downgrade the assertion to a warning.

Reply:
One more reason: With the `path` configured in a unix socket `address` , it seems that it's not picked up for consent screens, so I'll have to stay on the old config, until this can be resolved upstream ...

Reply:
summarizing in comment below